9-react.ts+TDD/src/App.test.tsx

```diff
@@ -0,0 +1,85 @@
+import faker from "faker";
+import puppeteer, { ElementHandle, Page, Browser } from "puppeteer";
```

Comment:
e2e tests should either sit in `e2e` folder or have `.e2e.spec.ts` extension (preferably both :) )